plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageAbstract.java

```diff
@@ -104,7 +104,7 @@ protected void createDriverPanel(Composite parent) {
 
         if (!site.isNew() && !site.getDriver().isEmbedded()) {
             Link netConfigLink = new Link(panel, SWT.NONE);
-            netConfigLink.setText("<a>Network settings (SSH, SSL, Proxy, ...)</a>");
+            netConfigLink.setText("<a>" + CoreMessages.dialog_connection_edit_wizard_conn_conf_network_link + "</a>");

```

Comment:
I'm not sure regarding this approach, for me the whole text (including "<a>" and "</a>") should go to resources

Reply:
Of course, I know what you say, but here the code <a> </a> is used for a link,  so I just translate the text.

Reply:
If you have any concern please refer to platform sources
'
package org.eclipse.debug.internal.ui.launchConfigurations;
...
public class SelectLaunchersDialog extends AbstractDebugListSelectionDialog {
...
	protected void addCustomHeaderControls(Composite parent) {
...
		Link link = new Link(comp, SWT.WRAP);
		link.setText(LaunchConfigurationsMessages.SelectLaunchersDialog_4);
...
'
and LaunchConfigurationsMessages.properties contains
'
SelectLaunchersDialog_4=<a href="ws">Change Workspace Settings...</a>
'

Reply:
Thanks for your example, then I know the standard practice.
Also thanks for your solution to the diff, I really learned.